test/system/src/test/java/io/pravega/test/system/MultiReaderTxnWriterWithFailoverTest.java

```diff
@@ -305,7 +303,30 @@ public void multiReaderTxnWriterWithFailOverTest() throws Exception {
         log.info("Test {} succeeds ", "MultiReaderWriterTxnWithFailOver");
     }
 
-    private void performFailoverTest() throws InterruptedException {
+    private void closeReadersAndWriters() {
+        log.info("Closing writers");
+        writerList.forEach(writer -> {
+            try {
+                writer.close();
+            } catch (RetriesExhaustedException e) {
+                log.warn("Unable to close the client: ", e);
```

Comment:
I think we should be strict. But I won't block the PR on that. So I'm approving it now so that it can get merged. (I hope with the change, but if both of you disagree, then leave it as is)